lib/ex_sdp/attribute/msid.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ExSDP.Attribute.Msid do
```

Comment:
```suggestion
defmodule ExSDP.Attribute.MSID do
```
also, can't it be `ExSDP.Media.MSID`?

Reply:
Acctually, why do we prefer MSID over Msid? Should we apply this pattern to Fmtp and Ssrc too?

Reply:
Regarding namespace, I've checked and there are attributes we can use both on session and media level e.g. `ice-ufrag` or `ice-pwd`.

Reply:
> Acctually, why do we prefer MSID over Msid? Should we apply this pattern to Fmtp and Ssrc too?

They're abbreviations, so this form is used everywhere. We also have RTP, SDP and VP8. So yes, I'd capitalize all of them.

> Regarding namespace, I've checked and there are attributes we can use both on session and media level e.g. ice-ufrag or ice-pwd.

Ok, so the only thing we can consider changing is `Attribute.X` -> `Attributes.X`, so that we could distinguish things like `Attribute.Parser` from `Attributes.MSID`

Reply:
At this moment modules `Attribute` and `Attribute.FMTP/SSRC/...` are almost the same. Both modules have some types and parse function. E.g. Attribute has `Attribute.framerate` or `Attribute.mid`. The only difference is that `parse` from `Attribute` can invoke `parse` from `Attribute.X`. If we want to rename `Attribute.X` to `Attributes.X` we probably should rename `Attribute` to `Attributes` too as `Attribute` is in fact aggregator for types that don't have their own structs. Do we want to do this?